include/zephyr/drivers/dma.h

```diff
@@ -229,6 +230,7 @@ struct dma_config {
  */
 struct dma_status {
 	bool busy;
+	bool enabled;
```

Comment:
Not used anywhere? Just added for future?

Reply:
Its a value that can be later read with dma_get_status() on SOF side. Right now there is no way to verify if a dma channel is enabled.